plutus-tx-plugin/src/PlutusTx/Compiler/Expr.hs

```diff
@@ -893,6 +903,7 @@ compileExpr e = traceCompilation 2 ("Compiling expr:" GHC.<+> GHC.ppr e) $ do
     GHC.Type _ -> throwPlain $ UnsupportedError "Types as standalone expressions"
     GHC.Coercion _ -> throwPlain $ UnsupportedError "Coercions as expressions"
 
+{- ORMOLU_DISABLE -}
```

Comment:
why?

Reply:
It was my quick way of working around this error:
```
$ fourmolu -i plutus-tx-plugin/src/PlutusTx/Compiler/Expr.hs
Loaded config from: /home/yura/projects/cardano/plutus/fourmolu.yaml
plutus-tx-plugin/src/PlutusTx/Compiler/Expr.hs:992:25-26
  The GHC parser (in Haddock mode) failed:
  [GHC-58481] parse error on input `<-'
```
When I removed the CPP conditional compilation block the problem went away. 
I'll try to update `fourmolu` to the latest version (14.1) to see if the problem is already solved.